Sources/Operators.swift

```diff
@@ -0,0 +1,181 @@
+//
+//  Operators.swift
+//  Relativity
+//
+//  Created by Dan Federman on 12/26/16.
+//  Copyright © 2016 Dan Federman.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import CoreGraphics
+import Foundation
+import UIKit
+
+
+// MARK: – ViewPosition Operators
+
+
+infix operator --> : AssignmentPrecedence
+infix operator <|> : AdditionPrecedence
```

Comment:
How would you feel about renaming `<|>` to `<>`?  Gets the same point across with fewer characters.